api_reference/source/api/evals.default_templates.rst

```diff
@@ -0,0 +1,7 @@
+evals.default\_templates module
```

Comment:
Is there ways to nix the 'module' here?

Reply:
The "module" and "package" in the names are removed before it's ultimately built through the filter_rst script!

Reply:
Yes, the pages won't build if not on the repo